src/http.js

```diff
@@ -25,6 +25,7 @@ export default function http(url, request) {
   if (request.requestInterceptor) {
     request = request.requestInterceptor(request) || request
   }
+  request.credentials = 'same-origin';
```

Comment:
moved

however I am not sure how to trigger the correct example in a unit test, hmm